projects/extension/src/containers/Popup.tsx

```diff
@@ -1,15 +1,82 @@
-import React, { FunctionComponent, useEffect, useRef, useState } from "react"
+import React, {
+  FunctionComponent,
+  ReactNode,
+  useEffect,
+  useRef,
+  useState,
+} from "react"
 
 import { MdOutlineSettings, MdCallMade } from "react-icons/md"
+import { BiDotsHorizontalRounded } from "react-icons/bi"
 
-import { Logo, ConnectedTab } from "../components"
+import { Accordion, ConnectedTab, Logo } from "../components"
 import { Background } from "../background"
-import { TabInterface } from "../types"
+
+const knownChains = ["polkadot", "kusama", "westend", "rococo"]
+
+interface PoupChains {
+  chainName: string
+  details?: ChainDetails[]
+}
+
+interface ChainDetails {
+  tabId: number
+  url: string
+  peers: number
+  isSyncing: boolean
+  chainId: string
+}
 
 const Popup: FunctionComponent = () => {
-  const disconnectTabRef = useRef<(tapId: number) => void>((_: number) => {})
-  const [activeTab, setActiveTab] = useState<TabInterface | undefined>()
-  const [otherTabs, setOtherTabs] = useState<TabInterface[]>([])
+  const disconnectTab = useRef<(tabId: number, chainId?: string) => void>(
```

Comment:
```suggestion
  const disconnectTab = useRef<(tabId: number) => void>(
```

Reply:
Also, why is this needed? Does calling `bg.uiInterface.disconnectTab()` not work?